applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -69,9 +74,9 @@ static size_t at_buf_len;
 static struct k_work cmd_send_work;
 static const char termination[3] = { '\0', '\r', '\n' };
 
-static u8_t uart_rx_buf[UART_RX_LEN];
+static u8_t uart_rx_buf[2][UART_RX_LEN];
```

Comment:
Please add a define for 2,make it possible to support more buffers later.
Also please consider a RX pointer associated with each buffer to separate interference.

Reply:
Defined UART_RX_BUF_NUM to 2 to state the number of buffers.